### PR TITLE
Updates webhook UUIDs to a new one when duplicating

### PR DESCRIPTION
**Pull request by krynble**
This PR fixes the issue mentioned https://github.com/n8n-io/n8n/issues/1967

Before duplicating the workflows, after the user confirms to save, we iterate over all nodes and check if they contain the `webhookId` property. In case they do, we replace it by a new one.

I am not 100% sure if this is the best approach (maybe I should've dispatched something to the store) but this worked.

### CODE REVIEW

File: packages/editor-ui/src/components/DuplicateWorkflowDialog.vue [hunk 1]
```diff
@@ -107,6 +109,13 @@ export default mixins(showMessage, workflowHelpers).extend({
 				return;
 			}
 
+			const allNodes = this.$store.getters.allNodes;
```
Comment (mutdmour) [inline packages/editor-ui/src/components/DuplicateWorkflowDialog.vue:112]: seems you are applying the changes before duplication happens, so you are editing the original workflow. if duplication fails, these changes will apply to the current workflow. 

You should move these changes into `saveAsNewWorkflow` and updating we are sending to the save endpoint

**Review by mutdmour — APPROVED** [review]
lgtm... just a couple of nitpicks

File: packages/editor-ui/src/components/mixins/workflowHelpers.ts [hunk 2]
```diff
@@ -435,13 +436,20 @@ export const workflowHelpers = mixins(
...
 					this.$store.commit('addActiveAction', 'workflowSaving');
 
 					const workflowDataRequest: IWorkflowDataUpdate = await this.getWorkflowDataToSave();
 					// make sure that the new ones are not active
 					workflowDataRequest.active = false;
+					if (updateWebhookUrls) {
+						workflowDataRequest.nodes!.forEach(node => {
+							if (node.webhookId) {
+								node.webhookId = uuidv4();
+							}
```
[the remark this answers is not in the document]
Comment (krynble) [reply packages/editor-ui/src/components/mixins/workflowHelpers.ts]: Thanks for the feedback! Name changed accordingly!

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with 0.135.0

**Pull request merged**